Client/Frontend/Browser/TopTabsViewController.swift

```diff
@@ -75,18 +76,24 @@ class TopTabsViewController: UIViewController {
         delegate.tabSelectionDelegate = self
         return delegate
     }()
-    
-    private weak var lastNormalTab: Tab?
-    private weak var lastPrivateTab: Tab?
-    
+
     private var tabsToDisplay: [Tab] {
         return self.isPrivate ? tabManager.privateTabs : tabManager.normalTabs
     }
-    
+
+    // Used for diffing collection view updates (animations)
+    private var isUpdating = false
+    private var pendingReloadData = false
+    private var _oldTabs: [Tab] = []
+    private weak var _oldSelectedTab: Tab?
```

Comment:
Confused: why is `_oldSelectedTab` weak, and `_oldTabs` not an array of weak references?

Nit: Hungarian naming.